NineChronicles.Headless/BlockChainService.cs

```diff
@@ -64,6 +68,14 @@ public UnaryResult<bool> PutTransaction(byte[] txBytes)
                 Transaction<PolymorphicAction<ActionBase>> tx =
                     Transaction<PolymorphicAction<ActionBase>>.Deserialize(txBytes);
 
+                var sentryTrace = SentrySdk.StartTransaction(
+                    tx.CustomActions[0].GetInnerActionTypeName() ?? "NoAction",
+                    "PutTransaction");
+                sentryTrace.SetTag("TxId", tx.Id.ToString());
```

Comment:
Extracting `tx.CustomActions[0].GetInnerActionTypeName()` and `tx.Id.ToString()` and declaring them as local variables may be better. 🤔